hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/HrsControllerBase.java

```diff
@@ -37,12 +42,22 @@
  * @author Eric Dalquist
  * @version $Revision: 1.2 $
  */
+@PropertySource("classpath:messages.properties")
+@Configuration
 public class HrsControllerBase {
     protected final Logger logger = LoggerFactory.getLogger(getClass());
     
     private String notificationPreferences = "notification";
     protected static final String helpUrlPreferences = "helpUrl";
+    private static final String GENERIC_ERROR_MESSAGE_PREFERENCE_NAME = "genericErrorMessage";
     private HrsUrlDao hrsUrlDao;
+    
+    private String defaultErrorMessage;
+    
+    @Bean
+    public static PropertySourcesPlaceholderConfigurer propertyConfigInDev() {
+        return new PropertySourcesPlaceholderConfigurer();
+    }
```

Comment:
This feels a little weird that `HrsControllerBase` is taking on the responsibility of publishing the `PropertySourcesPlaceholderConfigurer` bean to the `ApplicationContext`.  

Might be something to refactor away later.  I'm okay with shipping this now.